Content.Server/DetailExaminable/DetailExaminableComponent.cs

```diff
@@ -1,9 +1,11 @@
-﻿namespace Content.Server.DetailExaminable
-{
-    [RegisterComponent]
-    public sealed partial class DetailExaminableComponent : Component
-    {
-        [DataField("content", required: true)] [ViewVariables(VVAccess.ReadWrite)]
-        public string Content = "";
-    }
-}
+﻿// Parkstation-CharacterInformation-Start - This was moved to the Shared namespace
+// namespace Content.Server.DetailExaminable
+// {
+//     [RegisterComponent]
+//     public sealed class DetailExaminableComponent : Component
```

Comment:
Mark this as obsolete instead of commenting it to prevent merge conflict issues.